package.json

```diff
@@ -44,12 +44,13 @@
     "dependency:check": "dependency-check package.json --no-dev --missing ./dist/**/*.js",
     "test": "DOTENV_CONFIG_PATH=./.env.default jest --setupFiles dotenv/config -i",
     "version": "changeset version && npm install",
-    "changeset": "changeset"
+    "changeset": "changeset",
+    "run:v2": "OPENAPI_URL=https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v2.0/yaml/petstore.yaml npm run dev",
+    "run:v3": "OPENAPI_URL=https://raw.githubusercontent.com/OAI/OpenAPI-Specification/main/examples/v3.0/petstore.yaml npm run dev"
```

Comment:
My bad, I'm going to remove those before opening the PR. I'm using them to test both version of OpenAPI
```suggestion
```